mailpoet/assets/js/src/newsletters/send/sender-address-field.jsx

```diff
@@ -95,7 +95,7 @@ class SenderField extends Component {
   showSenderDomainError = (status) => {
     if (!status) return;
 
-    this.setState({ showSenderDomainWarning: true });
+    this.setState({ showSenderDomainWarning: status });
```

Comment:
I think this will still only work when `status` is `true` due to the early return above.

Reply:
I don't know what I was thinking...
I rewrote the commit.
I left this function as it was, and removed the `showSenderDomainError(false)` that was actually not needed.